engine/access/rpc/connection/manager.go

```diff
@@ -203,12 +210,19 @@ func (m *Manager) createConnection(address string, timeout time.Duration, cached
 	// The connections should be safe to be persisted and reused.
 	// https://pkg.go.dev/google.golang.org/grpc#WithKeepaliveParams
 	// https://grpc.io/blog/grpc-on-http2/#keeping-connections-alive
+	var opts []grpc.DialOption
+	opts = append(opts, grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(int(m.maxMsgSize))))
+	opts = append(opts, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	opts = append(opts, grpc.WithKeepaliveParams(keepaliveParams))
+	opts = append(opts, grpc.WithChainUnaryInterceptor(connInterceptors...))
+
+	if clientType == ExecutionClient && len(m.compressorName) > 0 {
```

Comment:
Can you also add compression support to collection nodes?
https://github.com/onflow/flow-go/blob/master/engine/collection/rpc/engine.go

They just have a `SendTransaction` endpoint

Reply:
```suggestion
	if m.compressorName != grpcutils.NoCompressor {
```